### PR TITLE
Added comment to `Brave New World`

### PR DESCRIPTION
**Pull request by RichardLitt**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -398,6 +398,8 @@ _Dystopian novels deal with imaginary communities or societies that are undesira
 
 #### [Brave New World](https://www.goodreads.com/book/show/5129.Brave_New_World) (1932) *by [Aldous Huxley](https://en.wikipedia.org/wiki/Aldous_Huxley)* [3.9] <img src="https://i.imgur.com/yJaoHfh.png" width="30px"> <img src="https://i.imgur.com/iA6WScw.png" width="30px">
 
+This book is insidiously horrifying in it's applicability, more so than _1984_ or _Fahrenheit 451_. [Here's a comic](https://abetterkuwait.files.wordpress.com/2011/08/orwell-huxley.jpg) that sums up the difference. Well worth the read. - [@RichardLitt](https://github.com/RichardLitt)
```
Comment (sindresorhus) [inline readme.md]: It's >>> its

Comment (RichardLitt) [inline readme.md]: God damn it.

Comment (RichardLitt) [inline readme.md]: Fixed.

**Comment by arthurvr** [comment]
Cool, thanks. lgtm

**Pull request merged**